src/key.rs

```diff
@@ -19,7 +19,12 @@ use crate::{ecdsa, SECP256K1};
 #[cfg(feature = "bitcoin_hashes")]
 use crate::{hashes, ThirtyTwoByteHash};
 
-/// Secret 256-bit key used as `x` in an ECDSA signature.
+/// Secret key - a 256-bit key used to create ECDSA and schnorr signatures.
+///
+/// A point on the secp256k1 curve i.e., a positive 256-bit integer guaranteed to be less than the
+/// secp256k1 curve order.
```

Comment:
A scalar isn't a point -- and while we often encode scalars as integers, mathematically they aren't integers, so I think describing them as such is a little misleading.

Reply:
Gee I botched that up. I removed the whole line. They aren't referred to as integers because we don't do integer math on them? Is it because they are members of a field, is that right? (Not adding as docs, just for my understanding.)

Reply:
@tcharding integers are in Z, scalars are in Z_n where n is the curve order.

Reply:
Cool, cheers.